developers/academy/zero_to_mvp/101_hello_weaviate/25_examples_2.mdx

```diff
@@ -111,13 +113,13 @@ Explore the world-famous Sydney Opera House and its incredible architecture! Fro
 
 </details>
 
-This is an example of a `generative search`, where Weaviate has retrieved information, and then leveraged a large language model (LLM) to re-shape it. This is a powerful feature that can transform how you deal with information.
+This process is an example of `generative search`. In a generative search, Weaviate retrieves information, and then leverages a large language model (LLM) to re-shape it. This is a powerful feature that can transform how you deal with information.
 
-You can vary the prompt to build any number of other outputs from this input.
+You can vary the prompt to build different kinds of output.
```

Comment:
Should this be `outputs` plural?

Reply:
Updated 

Output is an interesting word. I think it's like fish and fishes. Fish can be one or many of the same type, but fishes are many different types. I made "kind" plural because I wan't sure how to classify the output as different instance of the same type or different types.

I dodged it again with this update and went with "results" instead.